### PR TITLE
transform plane2 statuses back to v1

### PR DESCRIPTION
**Pull request by rolyatmax**

### CODE REVIEW

File: src/dev-server/plane.ts [hunk 1]
```diff
@@ -14,10 +15,19 @@ export type PlaneConnectResponse = {
   status_url: string
   status: string
 }
-export type PlaneStatusMessage = {
-  status: string,
-  time: number,
-}
+
+type PlaneTerminationReason = 'swept' | 'external' | 'keyexpired'
```
Comment (paulgb) [inline src/dev-server/plane.ts:19]: Hmm, should I make this `key_expired` on the Plane side before we do this?

**Review by paulgb — COMMENTED** [review]
Looks good, one question

File: src/dev-server/plane.ts (hunk 1 above)
Comment (rolyatmax) [reply src/dev-server/plane.ts:19]: I don't think we _have_ to wait for that. We aren't relying on "keyexpired" in this code, and this is also pinned to a version of Plane that formats it as "keyexpired". What do you think?

Comment (paulgb) [reply src/dev-server/plane.ts:19]: Ok, I can lgtm this and then separately make the Plane change.

**Review by paulgb — APPROVED** [review]

**Pull request merged**